### PR TITLE
Added detail about diesel_cli dependencies

### PR DESCRIPTION
**Pull request by ryanphillips838**
The added text comes directly from the diesel repo in the diesel_cli README.md.

### CODE REVIEW

File: source/views/guides/getting-started.html.slim [hunk 1]
```diff
@@ -74,7 +74,7 @@ main
           header.aside__header A Note on Installing <code>diesel_cli</code>
           .aside__text
             markdown:
-              If you run into an error like:
+              The diesel cli, by default, requires openssl, libpq, sqlite, and mysql. If you run into an error like:
```
Comment (weiznich) [inline source/views/guides/getting-started.html.slim:77]: I think we should be a bit more concrete about the sqlite and mysql dependencies here if we change that.
```suggestion
              The diesel cli, by default, requires openssl, libpq, libsqlite3, and libmysqlclient. If you run into an error like:
```

Beside of that I'm not really sold to adding `openssl` to this list as this is technically a sub dependency of `libpq` and you can theoretically build `libpq` without requiring `openssl` support.

Comment (ryanphillips838) [reply source/views/guides/getting-started.html.slim:77]: That entire line of text is copy and pasted from the readme file in the diesel_cli folder. If I remember right I couldn't install the cli without openssl. I actually like the idea of being more specific, this should definitely be in the getting started page somewhere either way. I put it in a place I thought it fit.

Comment (ryanphillips838) [reply source/views/guides/getting-started.html.slim:77]: I would also say openssl should stay there for people that don't want libpq on their machine.

Comment (weiznich) [reply source/views/guides/getting-started.html.slim:77]: I'm somewhat fine with having openssl it that list, but it should be clear that it is "required" for the postgres backend. For all other dependencies I think it is pretty clear which backend requires which dependency. Beside of that I'm not sure how to word this best to fit in the guide, so feel free to come up with a good suggestion :wink:

**Comment by weiznich** [comment]
Closed as there are merge conflicts after the website "rewrite"

**Pull request closed without merging**